docker/start.py

```diff
@@ -188,11 +188,6 @@ def main(args, environ):
     else:
         ownership = "{}:{}".format(desired_uid, desired_gid)
 
-    log(
```

Comment:
I've removed this while I was here because it looked like an error message, and I'm not sure what it was for (possibly debug?)

Reply:
I remember there being a problem with uid's in docker that a fix came in for. This was possibly debugging for it. Considering I haven't heard anyone speak of this problem for a while, it's probably unnecessary logging at this point.